api/serviceregistry/serviceregistry.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+
+package svreg;
+option go_package = "serviceregistry";
+
+
+service Registry{
+  rpc RegisterWorkload (ServiceWorkload) returns (Empty) {} // Request register of a new service
+  rpc QueryService (ServiceRequest) returns (ServiceWorkloads) {} // Request a services
+  rpc RemoveWorkload (ServiceWorkload) returns (Empty) {} // Remove service from list
+}
+
+message Empty {}
+
+//The request used to register a new service or update the existing ones
+message ServiceWorkload {
+  string serviceName = 1;
+  string connectivityDomain = 2;
+  Workload workload = 3;
+  repeated int32 ports = 4;
+}
+
+//Basic workload
+message Workload {
+  WorkloadIdentifier identifier = 1;
+  repeated string IPAddress = 2;
+  string connectivityDomain = 3;
```

Comment:
The connectivity domain information is present in both the Workload objects and the Service objects. Do we need it on both places?

Reply:
done